src/snapred/backend/dao/SmoothDataExcludingPeaksIngredients.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+from snapred.backend.dao.CrystallographicInfo import CrystallographicInfo
+from snapred.backend.dao.state.InstrumentState import InstrumentState
+
+
+class SmoothDataExcludingPeaksIngredients(BaseModel):
+    InstrumentState: InstrumentState
+    CrystalInfo: CrystallographicInfo
```

Comment:
Variable names, including member data, should begin with lowercase.  Names beginning with uppercase are for classes or constants.  So
```
class SmoothDataExcludingPeaksIngredients(BaseModel):
    instrumentState: InstrumentState
    crystalInfo: CrystallographicInfo
```

Reply:
Done.